src/vcpkg/postbuildlint.cpp

```diff
@@ -240,6 +240,28 @@ namespace vcpkg::PostBuildLint
         return LintStatus::SUCCESS;
     }
 
+    static LintStatus check_for_usage_forgot_install(const Filesystem& fs,
+                                                     const VcpkgPaths& paths,
+                                                     const Path& package_dir,
+                                                     const PackageSpec& spec)
+    {
+        static const std::string STANDARD_INSTALL_USAGE =
+            R"###(file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}"))###";
+
+        auto port_path = paths.root / "ports" / spec.name();
+        auto usage_path_from = port_path / "usage";
```

Comment:
And I don't know if we want to handle these cases:
<img width="351" alt="image" src="https://user-images.githubusercontent.com/41973254/215749900-99c7d956-d916-4f5b-838c-c9b26799414f.png">
So maybe check if there is a file in the port dir that contains usage and don't end with `.patch`

Reply:
But otherwise the usage is maybe only for msvc and should not be installed for linux and osx. Then this post build check would be wrong.

Reply:
True, maybe we should fatal to 'usage' and warning non-stop to other files contains 'usage'?